### PR TITLE
Add information about possible parent or child relationships in cim dto

### PR DESCRIPTION
**Pull request by thbaan**
## References

* #528

### CODE REVIEW

**Review by jonasdmoeller — DISMISSED** [review]

**Review by cmgrd — APPROVED** [review]

**Review by djorgensendk — DISMISSED** [review]

**Review by cmgrd — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ on:
 
 jobs:
   ci_base:
-    uses: Energinet-DataHub/.github/.github/workflows/ci-base.yml@2.1.0
+    uses: Energinet-DataHub/.github/.github/workflows/ci-base.yml@2.6.1
   
   dotnet_solution_ci:
     uses: Energinet-DataHub/.github/.github/workflows/dotnet-solution-ci.yml@2.0.0
```
Comment (dstsyst) [inline .github/workflows/ci.yml:27]: Suggestion: In the long run it might be simpler to update all reusable workflows to the same release

**Review by dstsyst — APPROVED** [review]
Only reviewed YML. Added one suggestion.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (thbaan) [reply .github/workflows/ci.yml:27]: Which are missing?

Comment (thbaan) [reply .github/workflows/ci.yml:27]: We will update all references to reusable workflows together next time.

**Pull request merged**